### PR TITLE
Added "minScore" +Fixed Typo "maxDetectionSize"

### PR DESCRIPTION
**Pull request by GiorgosXou**
_(Not used to pull requests but i found it important to make these tiny changes)_

### Added "minScore"
The minimum score of the returned bounding boxes of detected objects. Value between 0 and 1. Defaults to 0.5.

### Fixed Typo [*.md]
Fixed the typo "maxDetectionSize" to "maxNumBoxes"

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/432)

### CODE REVIEW

File: coco-ssd/src/index.ts [hunk 1]
```diff
@@ -209,13 +212,15 @@ export class ObjectDetection {
    * @param maxNumBoxes The maximum number of bounding boxes of detected
    * objects. There can be multiple objects of the same class, but at different
    * locations. Defaults to 20.
-   *
+   * @param minScore The minimum score of the returned bounding boxes of detected objects.
```
Comment (pyu10055) [inline coco-ssd/src/index.ts:216]: please reduce the length to less than 80 chars.

File: coco-ssd/src/index.ts [hunk 2]
```diff
@@ -101,11 +101,14 @@ export class ObjectDetection {
    * @param maxNumBoxes The maximum number of bounding boxes of detected
    * objects. There can be multiple objects of the same class, but at different
    * locations. Defaults to 20.
+   * @param minScore The minimum score of the returned bounding boxes of detected objects.
```
Comment (pyu10055) [inline coco-ssd/src/index.ts:105]: please reduce the length to less than 80 chars.

Comment (pyu10055) [reply coco-ssd/src/index.ts:105]: @GiorgosXou

File: coco-ssd/src/index.ts [hunk 3]
```diff
@@ -102,11 +102,14 @@ export class ObjectDetection {
...
```
Comment (pyu10055) [inline coco-ssd/src/index.ts:105]: Sorry one more change, This line is also over 80 characters.

Comment (GiorgosXou) [reply coco-ssd/src/index.ts]: My fault  😄

**Pull request merged**